src/Impostor.Api/Events/Game/Player/IPlayerWatchingCameraEvent.cs

```diff
@@ -0,0 +1,12 @@
+using Impostor.Api.Net.Inner.Objects;
+
+namespace Impostor.Api.Events.Player
+{
+    public interface IPlayerWatchingCameraEvent : IPlayerEvent
+    {
+        /// <summary>
+        ///     Gets the type of action - Started or Stopped watching
+        /// </summary>
+        bool Started { get; }
```

Comment:
Change to `IsWatching` or something

Reply:
Suggest `IsWatchingNow` and I will commit

Reply:
`IsWatching` +1

Reply:
resolved https://github.com/Impostor/Impostor/pull/197/commits/0f685d15791ba119ec16f81bea906f3feada211c